src/components/FeaturedPhotoSelector.jsx

```diff
@@ -1,29 +1,39 @@
 import React, { useState } from 'react';
-import { FormattedMessage } from 'react-intl';
 import { useTheme } from '@material-ui/core/styles';
 
 import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
 
-import CustomAlert from '../../../components/Alert';
-import usePatchSighting from '../../../models/sighting/usePatchSighting';
-import Button from '../../../components/Button';
-import StandardDialog from '../../../components/StandardDialog';
+import CustomAlert from './Alert';
+import usePatchSighting from '../models/sighting/usePatchSighting';
+import Button from './Button';
+import StandardDialog from './StandardDialog';
+import usePatchIndividual from '../models/individual/usePatchIndividual';
 
 export default function FeaturedPhotoSelector({
   open,
   onClose,
   assets,
-  sightingId,
+  sightingId = null,
+  individualId = null,
```

Comment:
`sightingId` and `individualId` will be undefined if they are not passed as properties on the props object. Why do you want to set them specifically to `null`?

Reply:
Didn't realize that I could optionally neglect props that didn't have default values. Cool!

Reply:
Done